client/src/components/Ratings.jsx

```diff
@@ -1,75 +1,48 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disable indent */
-
 import React from 'react';
+import { connect } from 'react-redux';
+import _ from 'underscore';
 import ReviewsList from './ReviewsList';
-// Don't have to import the server/index.js?
-
-
-
-// bring all reviews here to have data to go through
-
-// functions needed
-  // componentDidMount? Or alternative?
-
-  // calculate review to the nearest half star
-    // IMPORT Andrew's star function
-
-
-// calculate percentage of reviews recommend product
-// function getPercentageRecommended(results) {
-  // create new array of only 'true' values from results
-  // const trues = results.filter(
-  //   (resultItem.recommend) => (resultItem.recommend === true)
-  //     );
-
-  // return the Length of array of trues divided by original array length
-  // return Math.ceil(trues.length / ratingsArray.length) * 100;
-// }
 
+function getPercentageRecommended(results) {
+  const copy = results.map((result) => result.recommend);
+  const trues = copy.filter(
+    (resultItem) => (resultItem === true)
+  );
+  return Math.ceil(trues.length / results.length) * 100;
+}
 
-  // get average review value
-function getAverageReview(results) {
-  const reviewTotal = results.reduce(
+function averageToNearestTenth(array) {
+  const reviewVals = array.map(
+    (value) => value.rating
+  );
+  const reviewTotal = _.reduce(
+    reviewVals,
     (previousValue, currentValue) => previousValue + currentValue,
-    0
+    0,
   );
-
-  const { length } = results;
-  return (reviewTotal / length);
+  const number = (reviewTotal / array.length);
+  const whole = Math.floor(number);
+  const decimal = Math.trunc(((number % 1) * 10)) / 10;
+  return whole + decimal;
 }
```

Comment:
```javascript
function averageToNearestTenth(ratings) {
  const total = ratings.reduce(
    (total, item) => total + item.rating,
    0,
  );
  return (total / ratings.length).toFixed(1);
}
```

Reply:
Tweaked to: 
function averageToNearestTenth(ratings) {
  const total = ratings.reduce(
    (sum, item) => sum + item.rating,
    0,
  );
  return (total / ratings.length).toFixed(1);
}

But done.